packages/perennial/contracts/incentivizer/types/Program.sol

```diff
@@ -67,8 +67,10 @@ library ProgramLib {
         IOracleProvider.OracleVersion memory toOracleVersion = product.atVersion(versionComplete);
 
         uint256 inactiveDuration = programInfo.duration - (toOracleVersion.timestamp - fromOracleVersion.timestamp);
-        refundAmount = programInfo.amount.sum().muldiv(inactiveDuration, programInfo.duration);
+        UFixed18 refundAmount = programInfo.amount.sum().muldiv(inactiveDuration, programInfo.duration);
         self.available = self.available.sub(refundAmount);
+        address treasury = IIncentivizer(address(this)).treasury(programInfo.coordinatorId);
```

Comment:
can you use the existing treasury (product, programId) instead of added the new getting? I think you should have easy access to both of those here?

Reply:
oops i see